src/cogent3/core/new_alignment.py

```diff
@@ -2681,6 +2782,17 @@ def get_gapped_seq_bytes(
         step: OptInt = None,
     ) -> bytes: ...
 
+    @abstractmethod
+    def get_array_seqs(self, names: list[str]) -> numpy.ndarray:
```

Comment:
@GavinHuttley I don't remember this exact discussion? delete the abstract methods but keep the implementation on AlignedSeqsData?

Reply:
delete both the abstract methods and the method on AlignedSeqsData. Because a slice and strand information are only available to the Alignment and Aligned instances, those classes have to be the place for building `array_seqs` and `array_pos`.

Reply:
gotcha. done